src/test/java/io/jenkins/plugins/forensics/delta/model/ChangeTest.java

```diff
@@ -14,17 +14,28 @@
 class ChangeTest {
 
     private static final ChangeEditType EDIT_TYPE = ChangeEditType.INSERT;
+    private static final int CHANGED_FROM_LINE = 1;
+    private static final int CHANGED_TO_LINE = 1;
     private static final int FROM_LINE = 1;
     private static final int TO_LINE = 3;
 
     @Test
-    void testChangeGetters() {
+    void shouldHaveWorkingGetters() {
         Change change = createChange();
         assertThat(change.getEditType()).isEqualTo(EDIT_TYPE);
+        assertThat(change.getChangedFromLine()).isEqualTo(CHANGED_FROM_LINE);
```

Comment:
It would be more fluent if you could use the custom assertions. So:
```suggestion
        assertThat(change).hasChangedFromLine(CHANGED_FROM_LINE)
                .hasChangedToLine(...)
                ...;
```